Cargo.toml

```
@@ -66,11 +66,11 @@ once_cell = "1.17.1"
 sha2 = "0.10"
 serde = "1"
 serde_json = "1"
-# bump to a release after https://github.com/penumbra-zone/penumbra/pull/3598 is included
+# Corresponds with penumbra release 0.66
 # Note that when updating the penumbra versions, vendored types in `proto/sequencerapis/astria_vendored` may need to be updated as well.
-penumbra-ibc = { git = "https://github.com/penumbra-zone/penumbra.git", rev = "1b1be7e1373401eaf30d63352ee680991cb42aea", default-features = false }
-penumbra-proto = { git = "https://github.com/penumbra-zone/penumbra.git", rev = "1b1be7e1373401eaf30d63352ee680991cb42aea" }
-penumbra-tower-trace = { git = "https://github.com/penumbra-zone/penumbra.git", rev = "1b1be7e1373401eaf30d63352ee680991cb42aea" }
+penumbra-ibc = { git = "https://github.com/penumbra-zone/penumbra.git", rev = "29b93231c6dc4a534b7a1177790d3d39d2dfb52c", default-features = false }
```

Prefer the `tag` key. We used `rev` because we needed a not-yet (tag-)release feature:
```suggestion
penumbra-ibc = { git = "https://github.com/penumbra-zone/penumbra.git", tag = "v0.66.0", default-features = false }
```

And same for the others below